Packs/PrismaSaasSecurity/Integrations/SaasSecurityEventCollector/README.md

```diff
@@ -19,7 +19,7 @@ the challenges of:
     | Trust any certificate (not secure) | By default, SSL verification is enabled. If selected, the connection isn’t secure and all requests return an SSL error because the certificate cannot be verified. | False |
     | Use system proxy settings | Uses the system proxy server to communicate with the  integration. If not selected, the integration will not use the system proxy server. | False |
     | The maximum number of events per fetch. | The maximum number of events to fetch every time fetch is being executed. This number must be divisible by 100 due to Saas-Security api limitations. Default is 1000. In case this is empty, all available events will be fetched. | False |
-
+    | The maximum number of iterations to retrieve events. | In order to prevent timeouts, set this parameter to limit the number of iterations for retrieving event, Note - the default value is the recommended value to prevent timeouts. Default is 150. | False |
```

Comment:
```suggestion
    | The maximum number of iterations to retrieve events. | In order to prevent timeouts, set this parameter to limit the number of iterations for retrieving events. Note - the default value is the recommended value to prevent timeouts. Default is 150. | False |
```